frontend/packages/core/src/Input/text-field.tsx

```diff
@@ -1,37 +1,122 @@
-import React from "react";
-import type { InputLabelProps, TextFieldProps as MuiTextFieldProps } from "@material-ui/core";
+import * as React from "react";
+import styled from "@emotion/styled";
+import type {
+  InputProps as MuiInputProps,
+  StandardTextFieldProps as MuiStandardTextFieldProps,
+} from "@material-ui/core";
 import { TextField as MuiTextField } from "@material-ui/core";
-import styled from "styled-components";
+import ErrorIcon from "@material-ui/icons/Error";
+import _ from "lodash";
 
 const KEY_ENTER = 13;
-const StyledTextField = styled(MuiTextField)`
-  ${({ theme, ...props }) => `
-  display: flex;
-  width: 100%;
-  max-width: ${props["data-max-width"] || "500px"};
-  margin: 15px;
-  .MuiInputLabel-root {
-    color: ${theme.palette.text.primary};
-  }
-  .MuiInput-underline:after {
-    border-bottom: 2px solid ${theme.palette.accent.main};
-  }
-  `}
-`;
 
-export interface TextFieldProps {
-  maxWidth?: string;
+const BaseTextField = ({ InputProps, InputLabelProps, ...props }: MuiStandardTextFieldProps) => (
+  <MuiTextField
+    InputLabelProps={{ ...InputLabelProps, shrink: true }}
+    InputProps={{ ...InputProps, disableUnderline: true }}
+    fullWidth
+    {...props}
+  />
+);
+
+const StyledTextField = styled(BaseTextField)({
+  ".MuiInputLabel-root": {
+    fontSize: "13px",
+    fontWeight: "bold",
+    transform: "scale(1)",
+  },
+
+  ".MuiInputLabel-root, .MuiInputLabel-root.Mui-focused": {
+    color: "rgba(13, 16, 48, 0.6)",
+  },
+
+  ".MuiInputLabel-root.Mui-disabled": {
+    color: "rgba(13, 16, 48, 0.38)",
+  },
+
+  ".MuiInputLabel-root.Mui-error": {
+    color: "#db3615",
+  },
+
+  ".MuiInputBase-root": {
+    border: "1px solid rgba(13, 16, 48, 0.38)",
+    borderRadius: "4px",
+    fontSize: "16px",
+    padding: "14px 16px",
+  },
+
+  "label + .MuiInput-formControl": {
+    marginTop: "20px",
+  },
+
+  ".MuiInputBase-root.Mui-focused": {
+    borderColor: "#3548d4",
+  },
+
+  ".MuiInputBase-root.Mui-disabled": {
+    backgroundColor: "rgba(13, 16, 48, 0.12)",
+  },
+
+  ".MuiInput-input": {
+    padding: "0",
+    height: "20px",
+  },
+
+  ".MuiFormHelperText-root": {
+    verticalAlign: "middle",
+    display: "flex",
+    position: "relative",
+    fontSize: "12px",
+    marginTop: "7px",
+  },
+
+  ".MuiFormHelperText-root.Mui-error": {
+    color: "#db3615",
+  },
+
+  ".MuiInputBase-root.Mui-error": {
+    borderColor: "#db3615",
+  },
+
+  ".MuiFormHelperText-root > svg": {
+    height: "16px",
+    width: "16px",
+    marginRight: "5px",
```

Comment:
It looks like this is 4px in the designs

Reply:
the element in figma has 1px padding seen here, 1+4 = 5

![Screenshot from 2020-12-01 12-57-42](https://user-images.githubusercontent.com/4712430/100791557-046e6780-33df-11eb-86a1-3edcb27f7229.png)